akka-projection-core/src/main/scala/akka/projection/scaladsl/ProjectionManagement.scala

```diff
@@ -50,7 +51,7 @@ class ProjectionManagement(system: ActorSystem[_]) extends Extension {
   private def topic(projectionName: String): ActorRef[Topic.Command[ProjectionManagementCommand]] = {
     topics.computeIfAbsent(projectionName, _ => {
       val name = topicName(projectionName)
-      system.systemActorOf(Topic[ProjectionManagementCommand](name), name)
+      system.systemActorOf(Topic[ProjectionManagementCommand](name), sanitizeActorName(name))
```

Comment:
In the sample, we want to use the location id (`sweden/stockholm/kungsholmen`) as replica id, but that ends up being the projection name and to allow that we must make sure we can use whatever projection names as actor names everywhere where we do.